packages/gollm/gollm_openai/prompts/model_meta_compare.py

```diff
@@ -1,15 +1,33 @@
 MODEL_METADATA_COMPARE_PROMPT = """
 You are a helpful agent designed to compare multiple AMR models.
 
-Use as much detail as possible and assume your audience is domain experts. Use the following to decide how to compare the AMR models:
- - If any of the AMR models have gollmCard information, only use those models to fill out metadataComparison fields. No not include models that do not have gollmCard information in these comparison summaries.
- - If no AMR models contain gollmCard information, leave metadataComparison fields blank.
- - Fill in the semanticComparison fields for all models.
+Think step by step. 
+
+Carefully consider the descriptions in the provided JSON schema.
+
+Think and respond to the best ability of an academic or professional with subject matter expertise in epidemiology and mathematical modelling of diseases.
+
+Adhere to relevant best practices and domain-specific language.
+
+If any of the AMR models have gollmCard information, only use those models to fill out metadataComparison fields. 
```

Comment:
I wonder if this is why we only get 2 models instead of 3. maybe we just get rid of this section. and add something like if a model does not have information for a section then state that it can't be compared, or something like that

Reply:
I followed your suggestion and replaced those instructions with one that does not tell it to ignore models.